backend/webdav/webdav.go

```diff
@@ -159,6 +160,11 @@ Set to 0 to disable chunked uploading.
 			Help:     "Exclude ownCloud mounted storages",
 			Advanced: true,
 			Default:  false,
+		}, {
```

Comment:
Make this block a `var UnixSocketPathConfig = fs.Option{...}` in `fs/fshttp` and replace it here with just `fshttp.UnixSocketPathConfig`

Reply:
Done